build_tools/cmake/build_linux_benchmark_tools.sh

```diff
@@ -56,3 +56,30 @@ cd build-targets/linux-x86_64
 
 fi
 # --------------------------------------------------------------------------- #
+
+# --------------------------------------------------------------------------- #
+# Build for the target (linux-cuda).
+if [[ "${TARGETS}" == *"linux-cuda"* ]] || [[ "${TARGETS}" == *"all"* ]]
```

Comment:
Please follow Google bash style. Indentation, in particular would make this way more readable

https://google.github.io/styleguide/shellguide.html

Reply:
Done.